android/gradle.properties

```diff
@@ -18,13 +18,15 @@ org.gradle.jvmargs=-Xmx2048m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
 
+android.bundle.enableUncompressedNativeLibs=false
+
 # This one fixes a weird WebRTC runtime problem on some devices.
 # https://github.com/jitsi/jitsi-meet/issues/7911#issuecomment-714323255
 android.enableDexingArtifactTransform.desugaring=false
 
-android.useAndroidX=true
 android.enableJetifier=true
-android.bundle.enableUncompressedNativeLibs=false
+hermesEnabled=true
```

Comment:
Does this need to be here? Do libraries read it? If so I think we have react ext section, would it work there?